spec/Section 3 -- Type System.md

```diff
@@ -264,32 +264,43 @@ TypeDefinition :
 - EnumTypeDefinition
 - InputObjectTypeDefinition
 
-The fundamental unit of any GraphQL Schema is the type. There are six kinds of
+The fundamental unit of any GraphQL schema is the type. There are six kinds of
 named type definitions in GraphQL, and two wrapping types.
 
-The most basic type is a `Scalar`. A scalar represents a primitive value, like a
-string or an integer. Oftentimes, the possible responses for a scalar field are
-enumerable. GraphQL offers an `Enum` type in those cases, where the type
-specifies the space of valid responses.
+:: A _leaf type_ is a kind of type representing a primitive value which cannot
+be further selected and thus form the leaves in a response tree. GraphQL
```

Comment:
something does not match; if subject is 'kind' (singular) then the verb should be 'forms' (singular)

Reply:
General suggestion 
I think 'kind of type' is confusing, since we have already TYPE_KIND, different kind. What we introduce here are groups of types, sort of, purely for easier references in the spec; these groups or categories do not appear anywhere at runtime or in graphql artifacts - purely for shorter and simpler spec references.

So instead of 
  A Leaf type is a kind of type representing ...  

we can say simply 
  Leaf types are types representing

  without even explicitly calling it a group or category